addons/spreadsheet/static/src/pivot/plugins/pivot_ui_plugin.js

```diff
@@ -526,4 +559,5 @@ PivotUIPlugin.getters = [
     "getPivotDataSourceId",
     "getPivotTableStructure",
     "getPivotDomainArgsFromPosition",
+    "isPivotUnused",
```

Comment:
it would be nice to have some plugin tests here.
also, maybe add test with:
- references `ODOO.LIST(A1, ...)`  `ODOO.PIVOT(A1, ...)`
- ODOO.PIVOT.TABLE